easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.7.0-add_include_CheckSymbolExists_CMakeLists_txt.patch

```diff
@@ -0,0 +1,12 @@
+diff -Nru arpack-ng-3.7.0.orig/CMakeLists.txt arpack-ng-3.7.0/CMakeLists.txt
```

Comment:
Please add a description of this patch and your authorship.

Reply:
done in 3a7e150